src/rgw/rgw_sal_motr.cc

```diff
@@ -387,7 +388,18 @@ int MotrUser::store_user(const DoutPrefixProvider* dpp,
   RGWObjVersionTracker objv_tr = {};
   obj_version& obj_ver = objv_tr.read_version;
 
-  ldpp_dout(dpp, 10) << "Store_user(): User = " << info.user_id.id << dendl;
+  ldpp_dout(dpp, 20) << "Store_user(): User = " << info.user_id.id << dendl;
+  std::string access_key;
+  std::string secret_key;
+  if (!info.access_keys.empty()) {
+    std::map<std::string, RGWAccessKey>::const_iterator iter = info.access_keys.begin();
+    const RGWAccessKey& k = iter->second;
+    access_key = k.id;
+    secret_key = k.key;
+    MotrAccessKey MGWUserKeys(access_key, secret_key, info.user_id.id);
+    store->store_access_key(dpp, y, MGWUserKeys);
```

Comment:
Shouldn't we check if the access key exists already ?

Reply:
We are checking that while generating the access key.
Please refer to line 658 in [rgw_user.cc](https://github.com/Seagate/cortx-rgw/blob/dfb7127f0e0d560b0069379618fc00dc4fb26287/src/rgw/rgw_user.cc#L661)
A do while loop is checking whether a generated is already associated with some user.